NAME.md

```diff
@@ -0,0 +1,11 @@
+# Why is this project called "Factory Girl"?
+
+The name "Factory Girl" might be confusing to some developers who encounter this
+library.
+
+[We chose the name](https://robots.thoughtbot.com/waiting-for-a-factory-girl)
+as a reference to the [Rolling Stones song](https://www.youtube.com/watch?v=4jKix2DFlnA)
```

Comment:
How about putting the technical references first? Feels like the initial reason, and Rolling Stones second.